pandas/tests/io/test_sql.py

```diff
@@ -1665,6 +1665,29 @@ class Temporary(Base):
 
         tm.assert_frame_equal(df, expected)
 
+    def test_insert_multivalues(self):
```

Comment:
can you explicity test which backends support this.

Reply:
@jreback  I believe I've done this by adding a class variable to the below classes. Let me know if that addresses this concern